lib/src/collect.dart

```diff
@@ -497,3 +503,14 @@ class StdoutLog extends Log {
   @override
   void severe(String message) => print(message);
 }
+
+extension _ScopedOutput on Set<String> {
+  bool includesScript(String? scriptUriString) {
+    if (scriptUriString == null) return false;
+    if (isEmpty) return true;
```

Comment:
Shouldn't this return false if the set is empty? If not, can you leave a comment explaining why we're considering an empty set to include all scripts?

Reply:
Added a comment

Reply:
Thanks! That makes sense.